src/rgw/rgw_rest_conn.cc

```diff
@@ -21,7 +21,7 @@ RGWRESTConn::RGWRESTConn(CephContext *_cct, rgw::sal::Store* store,
     api_name(_api_name),
     host_style(_host_style)
 {
-  if (store) {
+  if (store && store->get_zone()) {
```

Comment:
@dang any idea how `get_zone()` would return null here? it seems reasonable to expect non-null here

Reply:
It shouldn't.  It's created on initialize() and never deleted.